indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -64,6 +64,8 @@ def _process(self):
         if (not enabled and self.field.type == RegistrationFormItemType.field_pd and
                 self.field.personal_data_type.is_required):
             raise BadRequest
+        positions = [item.position for item in self.regform.form_items if item.is_enabled == enabled]
```

Comment:
I think you could do this - you are looking for the highest position after all, aren't you?

    self.field.position = max(x.position for x in self.regform.form_items if item.is_enabled == enabled) + 1

However, see the comment below! max() on an empty collection would fail.